README.md

```diff
@@ -83,7 +83,7 @@ manager with the command `brew install --cask xquartz`.)
 on a computer with the Whitney family of fonts installed (specifically
 the Book, Medium, and Semibold variants). If installed on a computer
 *without* Whitney, the package will still work, but the fonts will
-default to Calibri (on Windows) or Arial (on macOS/Linux).
+default to the system's default sans-serif font (typically Arial).
```

Comment:
Will need to manually reknit the readme

Reply:
I tried to reknit with version 1.2, and the output code includes some familiar warnings, and the plots are using Arial.

```
#> Warning in grid.Call(C_textBounds, as.graphicsAnnot(x$label), x$x, x$y, : font
#> family not found in Windows font database
```

Reply:
Adding `dev = "ragg_png"` to the README knitr setup resolved the issue.